roles/redhat_release/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: initialize redhat_release facts
+  set_fact:
+    redhat_release_name: 'N/A (rpm not found)'
+    redhat_release_version: 'N/A (rpm not found)'
+    redhat_release_release: 'N/A (rpm not found)'
+
+- name: gather internal_redhat_release fact
+  raw: rpm -q --queryformat "%{NAME}\n%{VERSION}\n%{RELEASE}\n" --whatprovides redhat-release
+  register: internal_redhat_release
+  ignore_errors: yes
+  when: 'internal_have_rpm'
```

Comment:
Is it OK to rely on `internal_have_rpm` that is defined in another role (specifically, `check_dependencies`)? This _works_ but I believe only because we are already including the other role before this one.